### PR TITLE
fix: Dates - Out of Order Dec 11 <--> Dec 12

### PR DESCRIPTION
**Pull request by aaronholla**
Noticed this just now and was bugging me lol.

### CODE REVIEW

**Comment by tunnckoCore** [comment]
It's basically correct ordering :D Just the date was wrong, because I was tweaking my clock. Thanks any way.

File: README.md [hunk 1]
```diff
@@ -52,15 +52,15 @@ _Since [October 21, 2017](#october-21-2017)._ Hit :star: (_star_) for showing su
 
 
 
-### December 11, 2018 
+### December 12, 2018
```
Comment (tunnckoCore) [inline README.md:15]: You can remove that header at all. :)

**Comment by tunnckoCore** [comment]
Thanks!

**Comment by aaronholla** [comment]
Oh haha I see. No problem. This is an awesome list of resources, thank you!

**Pull request merged**